### DISCUSSION TITLE
개발자 계정 로그인 시, 계정 보호 지원 팝업창 관련 문의

### DISCUSSION THREAD

**Discussion by snowpeak1004** [fusiondevkr/pl900#43] [1 Q&A]
개발자 계정을 생성하여 최초 며칠은 정상적으로 로그인되는데.. 
며칠 후에는 계정보호지원이라는 팝업창이 뜹니다.  

14일 지나면 정상 로그인이 되지 않습니다. 어떻게 해야 하나요?

<img width="225" alt="계정보호지원" src="https://user-images.githubusercontent.com/87651237/130439773-1e364313-8cf0-430a-a8c4-b221027b3292.png">

**Comment by justinyoo** [comment]
2FA 를 설정하시면 됩니다.
